vdev/src/commands/build/mod.rs

```diff
@@ -0,0 +1,4 @@
+crate::cli_subcommands! {
+    "Build components..."
```

Comment:
Nit "components" may not be accurate for everything under here, IDK if I have a better term.